smart-contracts/contracts/interfaces/IPublicLock.sol

```diff
@@ -292,10 +292,12 @@ interface IPublicLock {
    * overtime.
    * @dev Throws if _owner does not have a valid key
    * @param _owner The owner of the key check the transfer fee for.
+   * @param _time The amount of time to calculate the fee for.
    * @return The transfer fee in basis-points(bps).
```

Comment:
Hmm the return units is not bps (?)

Reply:
I'm not sure returning bps makes sense now? The basis points give us a simple way to set the fee percentage, but when we want to calculate the fee, it seems like returning seconds would be the easiest to work with. That said, I don't feel strongly one way or the other.

Reply:
I'm going to proceed with modifying the "@return" comment to reflect the return unit in seconds.